mono/metadata/appdomain.c

```diff
@@ -2531,16 +2531,17 @@ static gsize WINAPI
 unload_thread_main (void *arg)
 {
 	MonoError error;
+	error_init (&error);
```

Comment:
So did we decide to initialize `MonoError` at declaration sites?

If we have, we should hold this PR until we update existing code.
If we haven't, this call to `error_init` is not needed.

In either case, I would prefer that *master* follow a single consistent pattern.

Reply:
I will undo these from this PR.